src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -35,7 +35,7 @@
                         <div class="nav-tabs-custom">
                             <ul class="nav nav-tabs" role="tablist">
                                 {% for name, form_tab in admin.formtabs %}
-                                    {% set _tab_name = ('tab_'~admin.code~'_'~loop.index)|replace({'.': '_'}) %}
+                                    {% set _tab_name = ('tab_'~admin.code~'_'~loop.index)|replace({'.': '_', '\\': '_'}) %}
```

Comment:
How about using `{{ _tab_name|e('html_attr') }}` instead? That would take care of that issue and of any other problematic characters.

Reply:
Something weird (I don't understand) happens: https://twigfiddle.com/zi3r0r

Reply:
@franmomu you lost a slash in inline declaration https://twigfiddle.com/h5aby2

Reply:
`e` means escaping, what's weird exactly?

Reply:
Sorry I didn't quite explained myself. Here https://twigfiddle.com/zi3r0r I expected to get the same result, but when there is a variable declared outside Twig it looks like it escapes it.

Using `{{ _tab_name|e('html_attr') }}` you get `tab_App&#x5C;YourBundle&#x5C;Admin&#x5C;PostAdmin_1` which is not a valid ID. I expected something like `tab_AppYourBundleAdminPostAdmin_1 `.

Reply:
Honest question : can a valid HTML attribute be an invalid ID? How do you know it's invalid?

Reply:
Sorry, talking about HTML attributes, it is as valid as `tab_App\YourBundle\Admin\PostAdmin_1` is, but that's not the problem. The problem is that is not valid if for tabs...

Reply:
I created this where you can see that: http://jsbin.com/sequhequyu/edit?html,output

Reply:
@greg0ire `{{ _tab_name|e('html_attr') }}` is not work solution

Reply:
Ok, I don't understand why it is not valid, but apparently it isn't.